pkg/controller/machineset/machineset_controller.go

```diff
@@ -175,8 +175,28 @@ func (c *MachineSetController) addMachineSet(obj interface{}) {
 
 func (c *MachineSetController) updateMachineSet(old, cur interface{}) {
 	ms := cur.(*clusteroperator.MachineSet)
+	oldms := cur.(*clusteroperator.MachineSet)
 	loggerForMachineSet(c.logger, ms).Debugf("enqueueing updated machine set")
 	c.enqueueMachineSet(ms)
+	if !oldms.Status.Installed && ms.Status.Installed {
```

Comment:
Is it fragile to only enqueue the other machinesets on the transition from not installed to installed? I think that I would rather just always enqueue all of the other machinesets when a master machineset is updated.

Reply:
I will switch it to check for master